### PR TITLE
ci: switch to codecov rather than codacy (#110)

### PR DESCRIPTION
**Pull request by holtgrewe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,8 @@
...
+[![codecov](https://codecov.io/gh/bihealth/altamisa/graph/badge.svg?token=R5GXUKCUYJ)](https://codecov.io/gh/bihealth/altamisa)
 [![Install with Bioconda](https://img.shields.io/badge/install%20with-bioconda-brightgreen.svg?style=flat)](http://bioconda.github.io)
 ![Continuous Integration Status](https://github.com/bihealth/altamisa/workflows/CI/badge.svg)
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/a853a56253604aa7ab87d2bcdcd9da51)](https://www.codacy.com/app/bihealth/altamisa?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=bihealth/altamisa&amp;utm_campaign=Badge_Grade)
-[![Coverage Badge](https://api.codacy.com/project/badge/Coverage/a853a56253604aa7ab87d2bcdcd9da51)](https://www.codacy.com/app/bihealth/altamisa?utm_source=github.com&utm_medium=referral&utm_content=bihealth/altamisa&utm_campaign=Badge_Coverage)
-[![Coverage Status](https://coveralls.io/repos/github/bihealth/altamisa/badge.svg?branch=master)](https://coveralls.io/github/bihealth/altamisa?branch=master)
+[![codecov](https://codecov.io/gh/bihealth/altamisa/graph/badge.svg?token=R5GXUKCUYJ)](https://codecov.io/gh/bihealth/altamisa)
```
Comment (sellth) [inline README.md:5]: duplicate

**Review by sellth — CHANGES_REQUESTED** [review]
We probably only need one Codedov badge.

File: README.md (hunk 1 above)
Comment (holtgrewe) [reply README.md:5]: done

**Review by sellth — APPROVED** [review]
LGTM

**Pull request merged**